modules/configuring-the-rhacs-operator-for-infrastructure-nodes.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_ocp/install-central-ocp.adoc
+// * cloud_service/installing_cloud_ocp/cloud-install-operator.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="configuring-the-rhacs-operator-for-infrastructure-nodes_{context}"]
+= Configuring the {product-title-short} Operator for infrastructure nodes
+
+To deploy the {rh-rhacs-first} Operator on infrastructure nodes, you can modify its subscription YAML to include `nodeSelector` and `tolerations` to ensure that its pods are scheduled correctly.
+
+.Procedure
+
+. Configure the {product-title-short} Operator with the necessary `nodeSelector` and `tolerations` for infrastructure nodes by using the following content, for example:
++
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  labels:
+    operators.coreos.com/rhacs-operator.rhacs-operator: ""
+  name: rhacs-operator
+  namespace: rhacs-operator
+spec:
+  channel: stable
+  config:
+    nodeSelector:
+      node-role.kubernetes.io/infra: "" # <1>
+    tolerations:
+      - effect: NoSchedule
+        key: node-role.kubernetes.io/infra # <2>
+        operator: Exists
+  installPlanApproval: Automatic
+  name: rhacs-operator
+  source: redhat-operators
+  sourceNamespace: openshift-marketplace
+----
++
+--
+<1> The `nodeSelector` key and value instruct the Kubernetes scheduler to place the Operator pods on nodes that have the label `node-role.kubernetes.io/infra: ""`.
+<2> The `tolerations` entry allows the Operator pods to be scheduled on nodes that have a taint with the `node-role.kubernetes.io/infra` key. This is necessary as infrastructure nodes are usually provided with a taint to prevent non-infrastructure workloads from being scheduled on them.
+--
+
+. Save the YAML file and apply it by using the OpenShift CLI (`oc`).
+
+When you apply the YAML file, you update the Subscription for the {product-title-short} Operator. This in turn configures where its pods can be scheduled. The `rhacs-operator` namespace is the designated location for the {product-title-short} Operator.
```

Comment:
```suggestion
When you apply the YAML file, you update the Subscription for the {product-title-short} Operator. This in turn configures where {product-title-short} Operator pods can be scheduled. The `rhacs-operator` namespace is the designated location for the {product-title-short} Operator.

To run {product-title-short} component pods on infrastructure nodes, you need to set the `nodeSelector` and `tolerations` fields for each component in their custom resources.
```

What I mean is, we should at least hint on the fact that this just takes care of the operator pod. More configuration is needed in the `Central` and/or `SecuredCluster` CRs to run the rest there.